### ISSUE TITLE
Does External Signals are working?

### ISSUE THREAD

**Issue by ng-designs** [akumidv/tradingview-assistant-chrome-extension#160]
Hi! Can you provide video or another guide to explain External Signals work? When i try example provided in extention, there is no markers on chart after loading file with next content:

```
timestamp,ticker,timeframe,signal
1701538511000,BTCUSDT,1h,BUY
1701531311000,BTCUSDT,1h,BUY
1701513311000,BTCUSDT,1h,BUY
1701502511000,BTCUSDT,1h,BUY
2023-10-27T01:00:00Z,BABA,1H,SELL
```

I just duplicated BTCUSDT string and changed time, generate it with https://www.epochconverter.com/

**Comment by akumidv** [comment]
Hi, probably late, for now priority is in refactoring code to change model of storing parameters and backtesting results to reduce memory usage.

**Comment by akumidv** [comment]
About marks. Do you set timeframe on chart to 1h? Also you need to add indicator `iondvSignals` on chart from example in extension or from repository https://github.com/akumidv/tradingview-assistant-chrome-extension/blob/main/pinescripts/iondv_signals.txt
After than - when you open indicator. The extension - set values to it and they should be showed on chart, If not - there is bug.

**Comment by ng-designs** [comment]
> About marks. Do you set timeframe on chart to 1h? Also you need to add indicator `iondvSignals` on chart from example in extension or from repository https://github.com/akumidv/tradingview-assistant-chrome-extension/blob/main/pinescripts/iondv_signals.txt After than - when you open indicator. The extension - set values to it and they should be showed on chart, If not - there is bug.

Yes, i've passed all steps from guide described in chrome extension, but it doesn't work. More than that, i've tested on MS Edge, Chromium and Chrome Portable browsers and still have the same result.
